src/OpenTelemetry.Instrumentation.AspNet/README.md

```diff
@@ -1,64 +1,159 @@
-# ASP.NET Instrumentation for OpenTelemetry .NET
+# ASP.NET Instrumentation for OpenTelemetry
 
 [![NuGet](https://img.shields.io/nuget/v/OpenTelemetry.Instrumentation.AspNet.svg)](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.AspNet)
 [![NuGet](https://img.shields.io/nuget/dt/OpenTelemetry.Instrumentation.AspNet.svg)](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.AspNet)
 
 Automatically instruments the incoming requests to
 [ASP.NET](https://docs.microsoft.com/aspnet/overview).
 
-## Installation
+## Steps to enable OpenTelemetry.Instrumentation.AspNet
+
+### Step1. Install Package
+
+Add a reference to the [`OpenTelemetry.Instrumentation.AspNet`](https://www.nuget.org/packages/opentelemetry.instrumentation.aspnet) package.
+Also, add any other instrumentations & exporters you will need.
 
 ```shell
 dotnet add package OpenTelemetry.Instrumentation.AspNet
 ```
 
-## Configuration
+### Step2. Modify Web.config
```

Comment:
Probably no need to put "Step #" here?

Reply:
I want to hear more opinions here.. Personally I like "Step 1", "Step 2". - something which makes it easier to refer to individual steps... (for eg: "Hey I have an issue about step 1". instead of saying "I have an issue about adding http module or web.config"..

Reply:
I don't have strong opinion on whether to keep the "step #".
I do have strong opinion that we should be consistent across this repo - if we need to add it, we want to do it across all the docs.

Reply:
> "Hey I have an issue about step 1"

Agree this is nice.

Small nit, maybe format `Step 1:` (space between "Step" and number + colon instead of period?)

Reply:
Yes once we get a generally agreed structure, I will make all docs consistent. (Alan agreed to help with docs as well)

(I still have to figure out how to write the readme.md for the OpenTelemetry package. Using that package alone is giving not much value unless clubbed with something else, but then that "something else" implicitly brings the OpenTelemetry, leading me to believe that standalone usage of OpenTelemetry package alone is a rare use case. Anyways let me discuss that in subsequent PRs)